pocs/camera/sbigudrv.py

```diff
@@ -162,7 +165,8 @@ def assign_handle(self, serial=None):
 
         # Serial number, name and type should match with those from Query USB Info obtained earlier
         camera_serial = str(self._camera_info.usbInfo[index].serialNumber, encoding='ascii')
-        assert camera_serial == ccd_info['serial_number'], self.logger.error('Serial number mismatch!')
+        assert camera_serial == ccd_info['serial_number'], self.logger.error(
```

Comment:
FWIW, this seems like an odd way to test things. assert isn't always executed, IIUC.